irods_client/__init__.py

```diff
@@ -0,0 +1 @@
+from .irodsClient import IrodsClient
```

Comment:
So the intention is to allow the following, correct?
```python
import irods_client
client = irods_client.IrodsClient(...)
```

Reply:
Just tested it out, and I had to do 

`from irods_client import IrodsClient
client = IrodsClient(...)`

I think both of these are better than what is there right now, since I included an unnecessary step. I do like having the imports be nice and short, though, so I'll try to figure out how to get it like that.

Reply:
Actually never mind, it does work either way, I just missed a mistake I made testing it out the first time.

Reply:
This isn't a blocker for the PR. We can address this in a separate PR. For now, open an issue so we don't forget to revisit it.

Reply:
Issue #27